pex/resolve/locked_resolve.py

```diff
@@ -148,7 +148,7 @@ def from_url(
 
     url = attr.ib()  # type: str
     fingerprint = attr.ib()  # type: Fingerprint
-    verified = attr.ib()  # type: bool
+    verified = attr.ib(eq=False)  # type: bool
```

Comment:
A comment here explaining why this is necessary would be good (and you mentioned adding one in your response to #2324).

Reply:
Ok, fixed. Now doing the right thing. Thanks for keeping me honest.